lighthouse/tests/main.rs

```diff
@@ -0,0 +1,3 @@
+mod account_manager;
```

Comment:
All these tests should probably run only in release mode too. I think moving the `#![cfg(not(debug_assertions))]` from `account_manager.rs` to `main.rs` should do it.